tests/business-logic/User.test.ts

```diff
@@ -0,0 +1,83 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let passwordHash: string;
+const email = "test@nueno.com";
+
+describe("User Entity", () => {
+  beforeEach(async () => {
+    await teardown();
+    passwordHash = await hashPassword("test@123.");
+  });
+
+  describe("#CreateNewUser", () => {
+    it("should create new user", async () => {
+      const body = {
+        name: "Chris Rock",
+        email,
+        password: passwordHash,
+      };
+      const userEntity = new UserEntity();
+      const result = await userEntity.create(body);
+      expect(result.message).toBe("User created");
+    }, 40000);
```

Comment:
I've had a similar problem with the timeouts, I'll find a solution for it that we don't need the timeout here. probably we'll need a local db instead.

the `40000` fix seems hacky. if you do hacky fixes please always add a comment why you did that, then it's easier to review. you can also add the comment to the Pull Request then the reviewer doesn't miss it.

If I wouldn't have had the same problem we would have wasted some time now because I would have been confused by the `40000`.